src/schema/rules/checks/asl.yaml

```diff
@@ -68,7 +68,7 @@ ASLFlipAngleASLContextLength:
     code: FLIP_ANGLE_NOT_MATCHING_ASLCONTEXT_TSV
     message: |
       The number of values for 'FlipAngle' for this file does not match the number of volumes in the
-      associated 'aslcontext.tsv'.
+      'sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_aslcontext.tsv'.
```

Comment:
Why are these being changed in this PR?

Reply:
Only the HED appendix should be changed here. I think this pull request has not merged previous recent BIDS changes.

Reply:
I tried merging `master` in, but this PR reverts recent changes. I've tried to pull out the relevant bits into #1623.